manifest-generation/config-from-cf-internal.yml

```diff
@@ -69,6 +74,8 @@ properties:
     server_key: (( merge ))
   loggregator:
     etcd:
+      ca_cert: (( merge || nil ))
+      require_ssl: (( merge || false ))
```

Comment:
Let's not duplicate the spec default: flowing a `null` through is a better practice, so this can just be `null`.